src/main/java/com/github/nicholasjgreen/dto/GenerateDataKeyResponse.java

```diff
@@ -1,15 +1,15 @@
-package DataKeyManagerIntegration.DTO;
+package com.github.nicholasjgreen.dto;
 
 @SuppressWarnings({"unused", "WeakerAccess"})
-public class GenerateDataKeyResult {
+public class GenerateDataKeyResponse {
```

Comment:
@jacsamell I originally called them response objects but then I was using them internally between providers and services. Should we still call them responses when they aren't only used as REST responses?